scripts/fetch-fonts.mjs

```
@@ -0,0 +1,193 @@
+        (fam.latinOnly ? `  (filtered from ${before})` : '')
+    );
+  }
+
+  // Download with bounded concurrency.
+  const entries = [...downloads.entries()];
+  let bytes = 0;
+  const LIMIT = 8;
+  for (let i = 0; i < entries.length; i += LIMIT) {
+    await Promise.all(
+      entries.slice(i, i + LIMIT).map(async ([local, url]) => {
+        const res = await fetch(url, { headers: { 'User-Agent': UA } });
```

ESLint

[error] 'fetch' is not defined.

(no-undef)